using-timescaledb/hypertables.md

```diff
@@ -106,29 +115,33 @@ corresponds to increased planning latency for some types of queries.
 both the underlying data size *and* any indexes, so some care might be
 taken if you make heavy use of expensive index types (e.g., some
 PostGIS geospatial indexes).  During testing, you might check your
-total chunk sizes via the [chunk relation size](#chunk_relation_size)
+total chunk sizes via the [chunk relation size][chunk_relation_size]
 function.
 
-**Space partitions**: The use of additional partitioning is a very
-specialized use case.  **Most users will not need to use it.**
+**Space partitions**: Space partitioning is optional but can make
+sense for certain types of data or when using distributed
+hypertables. In most cases, space partitioning does not offer any
+performance advantages for hypertables that are not distributed.
 
 Space partitions use hashing: Every distinct item is hashed to one of
-N buckets.  Remember that we are already using (flexible) time
-intervals to manage chunk sizes; the main purpose of space
-partitioning is to enable parallel I/O to the same time interval.
-
-Parallel I/O can benefit in two scenarios: (a) two or more concurrent
-queries should be able to read from different disks in parallel, or
-(b) a single query should be able to use query parallelization to read
-from multiple disks in parallel.
+N buckets. In a distributed hypertable, each bucket corresponds to a
+specific data node (although two or more buckets could map to the same
+node). Remember that we are already using (flexible) time intervals to
+manage chunk sizes; the main purpose of space partitioning is to
+enable parallel I/O to the same time interval, both within and across
+nodes.
+
+Parallel I/O can confer benefits in two scenarios: (a) two or more concurrent
+queries should be able to read from different nodes and/or disks in
+parallel, and (b) a single query should be able to use query
+parallelization to read from multiple nodes and/or disks in parallel.
```

Comment:
It's probably also worth mentioning that Parallel I/O can also greatly increase ingest rates when inserting data across several data nodes.